LDK/src/org/labkey/ldk/query/DefaultTableCustomizer.java

```diff
@@ -419,9 +420,9 @@ private static boolean configureMoreActionsBtn(TableInfo ti, List<ButtonConfigFa
                 btnNameMap.put(newButton.getText(), newButton);
                 menuItems.add(newButton);
 
-                for (ClientDependency cd : fact.getClientDependencies(ti.getUserSchema().getContainer(), ti.getUserSchema().getUser()))
+                for (Supplier<ClientDependency> cd : fact.getClientDependencies(ti.getUserSchema().getContainer(), ti.getUserSchema().getUser()))
                 {
-                    scripts.add(cd.getScriptString());
+                    scripts.add(cd.get().getScriptString());
```

Comment:
Consider resolving the `fact.getClientDependencies()` through `ClientDependency.getClientDependencySet()`. Main reason is to filter out nulls.